packages/slider/src/HandleController.ts

```diff
@@ -88,6 +88,21 @@ export class HandleController {
         }
         return result;
     }
+    private clickTimer: number | NodeJS.Timeout | null = null;
+
+    public get isDoubleClick(): boolean {
+        if (this.clickTimer !== null) {
+            clearTimeout(this.clickTimer);
+            this.clickTimer = null;
+            return true;
+        }
+
+        this.clickTimer = setTimeout(() => {
+            this.clickTimer = null;
+        }, 300);
```

Comment:
Just a note: PsWeb is using 500ms (as defined by Microsoft) to detect a double click:
see https://docs.microsoft.com/en-us/windows/win32/controls/ttm-setdelaytime?redirectedfrom=MSDN